pennylane/transforms/__init__.py

```diff
@@ -12,60 +12,89 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """
-This subpackage contains QNode, quantum function, device, and tape transforms.
+Quantum transforms are composable transformations of quantum functions. A
+quantum transform, when applied to a qfunc (a Python function containing
+quantum operations) or a :class:`~pennylane.QNode`, will **transform** the
+quantum circuit in order to achieve the desired result. For example, quantum
+transforms can be used to:
 
+- extract information from the quantum function (e.g., to extract unitary
+  matrices or draw the underlying circuit),
+
+- modify or compile the quantum function,
+
+- generate new quantum functions that compute quantum properties and metrics,
+  such as the gradient and the Fisher information matrix.
+
+This module provides a selection of device-independent, differentiable quantum
+gradient transforms. As such, not only is the output of a quantum transform
+differentiable, but quantum transforms *themselves* can be differentiated with
+respect to any floating point arguments.
+
+In addition, this module also includes an API for writing your own quantum
+transforms.
+
+.. note::
+
+    For gradient transforms, please see the :mod:`qml.gradients <pennylane.gradients>`
+    module.
 
 .. currentmodule:: pennylane
 
-Transforms
-----------
+Overview
+--------
 
-Transforms that act on QNodes
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Information transforms
+~~~~~~~~~~~~~~~~~~~~~~
 
 These transforms accept QNodes, and return new transformed functions
-that compute the desired quantity.
+that compute the desired quantity. These transforms do not result
+in quantum executions.
 
 .. autosummary::
     :toctree: api
 
-    ~transforms.classical_jacobian
-    ~batch_params
     ~draw
-    ~transforms.get_unitary_matrix
-    ~metric_tensor
     ~specs
-    ~transforms.mitigate_with_zne
+    ~transforms.get_unitary_matrix
+    ~transforms.classical_jacobian
 
-Transforms that act on quantum functions
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-These transforms accept quantum functions (Python functions
-containing quantum operations) that are used to construct QNodes.
+Multiple circuit transforms
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
I'm not very happy with this category name/description, but can't think of anything better 🤔

Reply:
```suggestion
Batch circuit transforms
~~~~~~~~~~~~~~~~~~~~~~~~
```